app/controllers/case_studies_controller.rb

```diff
@@ -91,9 +106,24 @@ def update
   def create
     @case_study = CaseStudy.new(params[:case_study])
 
+    if Rails.env.production?
+      file = params[:case_study][:picture]
```

Comment:
Probably better to refactor some/all of this method from the controller to the model to keep with the rails convention of simple controllers when possible.